packages/boxel-ui/addon/src/components/resizable-panel-group/index.gts

```diff
@@ -218,6 +217,17 @@ export default class ResizablePanelGroup extends Component<Signature> {
           context.defaultLengthFraction * this.panelGroupLengthPx;
       }
     }
+    //Update previous lengthPx
+    let previousId = id - 1;
+    let previousContextEl = this.listPanelContext.get(previousId);
+    if (
+      previousContextEl !== undefined &&
+      previousContextEl.defaultLengthFraction &&
+      this.panelGroupLengthPx
+    ) {
+      previousContextEl.lengthPx =
+        previousContextEl.defaultLengthFraction * this.panelGroupLengthPx;
+    }
```

Comment:
The issue here is when you add a panel, the last remaining panel which stayed but didn get unregistered will have `panelGroupLengthPx = <the entire width of panel group size>`. This lengthPx essentially becomes stale when you  register a new panel -- hence, the screen becomes warped (bigger) in the left-panel